src/FSharp.Data.GraphQL.Shared/WebSockets.fs

```diff
@@ -35,7 +38,7 @@ type ServerMessage =
     | ConnectionAck
     | ServerPing
     | ServerPong of JsonDocument voption
-    | Next of id : string * payload : Output
+    | Next of id : string * payload : SubscriptionExecutionResult
     | Error of id : string * err : NameValueLookup list
     | Complete of id : string
```

Comment:
Also it would be nice to make these DUs serializable right away using 
`[<JsonFSharpConverter(unionTagName = "type")>]`
This will eliminate the need of `serializeServerMessage` function

Reply:
For each union case you will need to apply attributes like `[<JsonName "connection_ask">]` then